extensions/wikia/RTE/RTE_setup.php

```diff
@@ -7,21 +8,22 @@
 	'author' => array('Inez Korczyński', 'Maciej Brencz')
 );
 
-$dir = dirname(__FILE__);
+
 
 // autoloaded classes
-$wgAutoloadClasses['RTE'] = "$dir/RTE.class.php";
-$wgAutoloadClasses['RTEAjax'] = "$dir/RTEAjax.class.php";
-$wgAutoloadClasses['RTEData'] = "$dir/RTEData.class.php";
-$wgAutoloadClasses['RTELang'] = "$dir/RTELang.class.php";
-$wgAutoloadClasses['RTELinkerHooks'] = "$dir/RTELinkerHooks.class.php";
-$wgAutoloadClasses['RTEMagicWord'] = "$dir/RTEMagicWord.class.php";
-$wgAutoloadClasses['RTEMarker'] = "$dir/RTEMarker.class.php";
-$wgAutoloadClasses['RTEParser'] = "$dir/RTEParser.class.php";
-$wgAutoloadClasses['RTEReverseParser'] = "$dir/RTEReverseParser.class.php";
-$wgAutoloadClasses['RTEController'] = "$dir/RTEController.class.php";
+$wgAutoloadClasses['RTE'] = __DIR__ . '/RTE.class.php';
+$wgAutoloadClasses['RTEAjax'] = __DIR__ . "/RTEAjax.class.php";
+$wgAutoloadClasses['RTEData'] = __DIR__ . "/RTEData.class.php";
+$wgAutoloadClasses['RTELang'] = __DIR__ . "/RTELang.class.php";
+$wgAutoloadClasses['RTELinkerHooks'] = __DIR__ . "/RTELinkerHooks.class.php";
+$wgAutoloadClasses['RTEMagicWord'] = __DIR__ . "/RTEMagicWord.class.php";
+$wgAutoloadClasses['RTEMarker'] = __DIR__ . "/RTEMarker.class.php";
+$wgAutoloadClasses['RTEParser'] = __DIR__ . "/RTEParser.class.php";
+$wgAutoloadClasses['RTEReverseParser'] = __DIR__ . "/RTEReverseParser.class.php";
+$wgAutoloadClasses['RTEController'] = __DIR__ . "/RTEController.class.php";
```

Comment:
Minor issue: we can use single quotes here. Or, in other words, we don't need double quotes here. :-)